cas/service.c

```diff
@@ -1326,6 +1373,35 @@ static int cas_fom_tick(struct m0_fom *fom0)
 			m0_ctg_op_init(&fom->cf_ctg_op, fom0,
 				       cas_op(fom0)->cg_flags);
 		}
+
+		/* If cas request has a txr payload, this a dtm0 operation.
```

Comment:
s/this a dtm0 operation/this is a dtm0 operation

Reply:
Done.